app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -39,6 +41,11 @@ export default Component.extend(LoadMore, {
     return order === "likes";
   },
 
+  @discourseComputed("selected")
+  canDoBulkActions(selected) {
+    return this.currentUser?.staff && selected;
+  },
+
```

Comment:
(This could be `canDoBulkActions: and("currentUser.staff", "selected"),` but I leave it up to you 😉)